src/MujocoSimulator.lf

```diff
@@ -46,24 +39,25 @@ reactor MujocoSimulator(
     state lastx: double = 0;
     state lasty: double = 0;
     
-    output model_pointer: mjModel*;
-    output call_controller: mjData*;
+    // Use typedef'ed pointers to avoid C-runtime mem management.
```

Comment:
Is your goal here trying to bypass the token mechanism in the C runtime?

Reply:
Yes, this fixes a double free error I observe

Reply:
This is a great fix! I was scratching my head over this double free error...